dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java

```diff
@@ -86,8 +90,18 @@ public static Path generate()
 
     ArtifactCache cache = loadArtifactInfo(managedDependencies);
     
+    ImmutableList<Path> classpath = StaticLinkageChecker.artifactsToClasspath(managedDependencies);
+    // TODO(suztomo): to take command-line option to choose entry point classes for reachability
```

Comment:
Wait are you leaving a TODO for someone else without them being a reviewer on the PR?

Reply:
Is this relevant for the dashboard? It doesn't seem like it would be.

Reply:
rewrote